docs/interpreter/jdbc.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: page
+title: "Generic JDBC Interpreter"
+description: "JDBC user guide"
+group: manual
+---
+{% include JB/setup %}
+
+
+## Generic JDBC  Interpreter for Apache Zeppelin
+
+This interpreter lets you create a JDBC connection to any data source, by now it has been tested with:
+
+* Postgres
+* MySql
+* MariaDB
+* Redshift
+* Hive
+
+If someone else used another database please report how it works to improve functionality.
+
+### Create Interpreter
+
+When create a interpreter by default use PostgreSQL with the next properties:
+
+<table class="table-configuration">
+  <tr>
+    <th>name</th>
+    <th>value</th>
+  </tr>
+  <tr>
+    <td>common.max_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>default.driver</td>
+    <td>org.postgresql.Driver</td>
+  </tr>
+  <tr>
+    <td>default.password</td>
+    <td>`********`</td>
+  </tr>
+  <tr>
+    <td>default.url</td>
+    <td>jdbc:postgresql://localhost:5432/</td>
+  </tr>
+  <tr>
+    <td>default.user</td>
+    <td>gpadmin</td>
+  </tr>      
+</table>
+
+Is not necessary add jar driver to the classpath for PostgreSQL.
+
+#### Simple connection
+
+Before creating the interpreter it is necessary to add to the Zeppelin classpath the path of the JDBC you want to use, to do it you must edit the file `zeppelin-daemon.sh` as shown in the table:
+
+```
+# Add jdbc connector jar
+ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/mysql-connector-java-5.1.6.jar"
+```
+
+For create the interpreter you need to specify connection parameters as shown in the table.
+
+<table class="table-configuration">
+  <tr>
+    <th>name</th>
+    <th>value</th>
+  </tr>
+  <tr>
+    <td>common.max_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>{prefix}.driver</td>
```

Comment:
should an user replace this `{prefix}` then? Does he need to do that even when there is only one jdbc interpreter? Would it be clearer to just have "default.driver" here for the simple connection case here?